_docs-sources/guides/stay-up-to-date/cis/cis-1.5.0/deployment-walkthrough/step-2-update-references-to-the-gruntwork-infrastructure-as-code-library.md

```diff
@@ -0,0 +1,256 @@
+---
+sidebar_label: Update references to the Gruntwork Infrastructure as Code Library
+---
+
+# Step 2: Update references to the Gruntwork Infrastructure as Code Library
+
+To update to the CIS AWS Foundations Benchmark v1.5.0, you need to update your references to the Gruntwork
+Infrastructure as Code Library to use compatible versions. We (Gruntwork) have reviewed and updated all the library modules for compatibility with the new version of the benchmark. As a customer, you need to update to the correct versions of the Gruntwork library to pick up the changes necessary for your infrastructure to be compatible.
+
+The following table provides a summary of all the relevant Gruntwork AWS modules and the respective versions that are
+compatible with CIS AWS v1.5.0:
```

Comment:
Put the Patcher mention _first_. Put the table later, into the "manual" portion! Otherwise, you're burying the lede, and I bet some customers will start manually updating before reading about Patcher further down the page.